api/v1alpha1/database_types.go

```diff
@@ -35,8 +35,9 @@ type DatabaseSpec struct {
 	// These keywords can be used: Protocol, DatabaseHost, DatabasePort, UserName, Password, DatabaseName.
 	// Default template looks like this:
 	// "{{ .Protocol }}://{{ .UserName }}:{{ .Password }}@{{ .DatabaseHost }}:{{ .DatabasePort }}/{{ .DatabaseName }}"
-	ConnectionStringTemplate string   `json:"connectionStringTemplate,omitempty"`
-	Postgres                 Postgres `json:"postgres,omitempty"`
+	ConnectionStringTemplate string            `json:"connectionStringTemplate,omitempty"`
+	SecretsTemplates         map[string]string `json:"secretsTemplates,omitempty"`
```

Comment:
think we should also write a readme for this feature, or?

Reply:
I've updated docs here: https://github.com/kloeckner-i/db-operator/blob/155-secrets-templating/docs/creatingdatabases.md